<launch>

  <node name="recognizer" pkg="pocketsphinx" type="recognizer.py">
    <param name="lm" value="$(find Voice_interaction)/config/text.lm"/>
    <param name="dict" value="$(find Voice_interaction)/config/text.dic"/>
  </node>

  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>

  <node name="voice_interact" pkg="Voice_interaction" type="voice_interact.py" output="screen">
    <param name="voice" value="voice_don_diphone"/>
    <param name="wavepath" value="$(find pi_speech_tutorial)/sounds"/>
  </node>

</launch>
